src/com/example/ppj05/Task01.java

```diff
@@ -0,0 +1,37 @@
+package com.example.ppj05;
+import java.util.Scanner;
+public class Task01 {
+
+    public static void main(String[] args) {
+        // Input three numbers from the keyboard.
```

Comment:
Please remove redundant comments. 
And I remember that this task was already done in the other PR.